vulcanai/models/metrics.py

```diff
@@ -61,6 +61,9 @@ def get_score(targets, predictions, metrics='accuracy', average=None,
         """
         double_parametered_functions = ["get_accuracy"]
 
+        if isinstance(targets, torch.Tensor):
+            targets = targets.cpu().numpy()
```

Comment:
this should be `targets = targets.cpu().detach().numpy()`

Reply:
👍